math/maxima/Portfile

```diff
@@ -19,30 +18,62 @@ long_description \
         tensors. Maxima yields high precision numeric results \
         by using exact fractions, arbitrary precision integers, \
         and variable precision floating point numbers. Maxima \
-        can plot functions and data in two and three dimensions.
+        can plot functions and data in two and three dimensions. \n\
+        Maxima includes the following commands: \n\
+        * \"rmaxima\" is a front end which provides advanced line-editing \
+        facilities via rlwrap. \n\
```

Comment:
No `\n`s in long_description, please.

Reply:
@l2dy It seems other people do that too and it's harmless.

Reply:
@pmetzger OK, you convinced me. But @tomio-arisaka please remove trailing spaces before newline (`\n`).

Reply:
I will remove '\n' in log_description.